documentation/templates/privilege_escalation.md.erb

```diff
@@ -99,3 +99,63 @@ This enables using any executable and command-line options to change users. Keep
 - Bolt always appends the specified `run-as` user to the specified `run-as-command`,
   which can be limiting for some escalating commands.
 - Bolt will not use the `run-as-command` option if `run-as` is not set.
+
+## Escalating privilege over WinRM
+
+As you may have noted we do not support `run-as` for the WinRM transport. This is because the way
+WinRM works is by connecting with the user that has permissions to perform the actions, not by
+elevating or changing to another user after connection. WinRM operates with a session level token
+and has no functionality to change to another user after authentication. Instead, you can connect
+with one user, then provide credentials to a program that accepts them to perform another action,
+but that is not part of WinRm.
```

Comment:
```suggestion
The WinRM transport does not support `run-as`. This is because
WinRM operates with a session level token and has no functionality to change to another user after
authentication. Once you connect with a user, you can only perform actions that you have permission to
perform with that user.  Instead of switching users, you can connect with a user, and then provide credentials
to a program that accepts them to perform another action. The behavior of providing credentials to a 
program is not part of WinRM.
```